index.js

```diff
@@ -26,7 +26,6 @@ module.exports.Comment = require('./lib/comment');
 
 // Compatibility synonyms
 Document.fromXmlString = Document.fromXml;
-Document.fromHtmlString = Document.fromHtmlString;
```

Comment:
Yea, sorry, this is what I meant. I think the current line was a typo and was meant to be what you have written.